core/commands/dag/dag.go

```diff
@@ -48,6 +50,7 @@ into an object of the specified format.
 		cmds.StringOption("format", "f", "Format that the object will be added as.").Default("cbor"),
 		cmds.StringOption("input-enc", "Format that the input object will be.").Default("json"),
 		cmds.BoolOption("pin", "Pin this object when adding.").Default(false),
+		cmds.StringOption("hash", "Hash function to use").Default(""),
```

Comment:
It's `hash` in `ipfs add`: https://github.com/ipfs/go-ipfs/blob/master/core/commands/add.go#L44, I guess it's better for consistency